Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -7049,6 +7050,13 @@ - (void)_elementDidFocus:(const WebKit::FocusedElementInformation&)information u
             [self.window makeKeyWindow];
     }
 
+    if (!UIKeyboard.activeKeyboard) {
```

Comment:
Just to confirm, `UIKeyboard.activeKeyboard` does not imply anything about visibility state right?

(i.e. the keyboard can be hidden but `UIKeyboard.activeKeyboard` is non-nil)

Reply:
Yeah, that's correct — though, the intent was really to detect this "non-UI application" and avoid waiting for the keyboardWillShow notification that will never arrive.

I'll poke around a bit more and see if there's a cleaner way to achieve this...

Reply:
I think this is fine – was just concerned given the `active` in the name of the method.